packages/components/src/components/Select/useSelectData.tsx

```diff
@@ -0,0 +1,82 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { IconCross } from '@frontify/fondue-icons';
+import { Children, cloneElement, isValidElement, useMemo, useState, type ReactNode } from 'react';
+
+import { ForwardedRefSelectItem, type SelectItemProps } from './SelectItem';
+import { ForwardedRefSelectSlot, type SelectSlotProps } from './SelectSlot';
+import { getSelectOptionValue } from './utils';
+
+export type SelectItemType = {
+    value: string;
+    label: string;
+};
+
+export const getRecursiveOptionValues = (children: ReactNode): { value: string; label: string }[] => {
```

Comment:
Please add some utility tests for both utility and hook under

Reply:
added tests

Reply:
same here, I don't see the Vitest test file